File: engine/b/src/main/java/net/crazyinvestor/engine_b/event/BithumbOrderbookDepthEvent.java
package net.crazyinvestor.engine_b.event;

import net.crazyinvestor.engine_b.dto.response.BithumbWSOrderbookDepthResponse;
import org.springframework.context.ApplicationEvent;

public final class BithumbOrderbookDepthEvent extends ApplicationEvent {
    public BithumbOrderbookDepthEvent(BithumbWSOrderbookDepthResponse response) {
        super(response);
    }
}

File: engine/a/ticker_producer_engine/src/main/kotlin/net/crazyinvestor/ticker_producer_engine/event/BithumbOrderbookDepthEvent.kt
package net.crazyinvestor.ticker_producer_engine.event

import net.crazyinvestor.ticker_producer_engine.dto.response.BithumbWSOrderbookDepthResponse
import org.springframework.context.ApplicationEvent

class BithumbOrderbookDepthEvent(response: BithumbWSOrderbookDepthResponse) : ApplicationEvent(response)